domain/lib/src/repository/document/document_repository.dart

```diff
@@ -54,4 +55,6 @@ abstract class DocumentRepository {
   Future<Uri> downloadDocumentIOS(Document document, Token token, Uri baseUrl, CancelToken cancelToken);
 
   Future<Document> remove(DocumentId documentId);
+
+  Future<List<Document>> copyToMySpace(CopyRequest copyRequest);
```

Comment:
why copy 1 file, but the return is List of document?

Reply:
My bad, I took android version for example but yes it seems the API returns only one doc (cf API doc)

Reply:
And my bad again after investigation API returns a list !